src/VolleyManagement.Backend/VolleyManagement.Services/TournamentService.cs

```diff
@@ -343,7 +344,7 @@ public void AddTeamsToTournament(List<TeamTournamentAssignmentDto> groupTeam)
                     TournamentResources.CollectionIsEmpty);
             }
 
-            var tournamentId = GetTournamentByGroup(groupTeam[0].GroupId).Id;
+            var tournamentId = GetTournamentByGroup(((List<TeamTournamentAssignmentDto>)(groupTeam))[0].GroupId).Id;
```

Comment:
What is going on here?

Reply:
![image](https://user-images.githubusercontent.com/17709640/36745255-973ac80e-1bf7-11e8-8472-610fd7899c58.png)
strange things it require to make it ICollection and the code reqires to get [0]from list,
I can write in signature List but it will add 2 warnings of type s3956

Reply:
Any other ideas on how to fix that?

Reply:
Це ніяк не вплине на користувача, тому нехай і буде

Reply:
Then I really suggest you to learn this programming language https://en.wikipedia.org/wiki/Brainfuck

Reply:
Cast is necessary, not here, that in another snippet